archetype/engine-v2/src/main/java/io/helidon/build/archetype/engine/v2/ast/Input.java

```diff
@@ -692,6 +790,101 @@ public String toString() {
         }
     }
 
+    /**
+     * List of Validation.
+     */
+    public static final class Validations extends Input {
+
+        private final String id;
+        private final String description;
+
+        private Validations(Input.Builder builder) {
+            super(builder);
+            this.id = builder.attribute("id", true).asString();
+            this.description = builder.attribute("description", false).asString();
+        }
+
+        @Override
+        public <A> VisitResult accept(Input.Visitor<A> visitor, A arg) {
+            return visitor.visitValidations(this, arg);
+        }
+
+        @Override
+        public <A> VisitResult acceptAfter(Input.Visitor<A> visitor, A arg) {
+            return visitor.postVisitValidations(this, arg);
+        }
+
+        /**
+         * Get the id.
+         *
+         * @return id
+         */
+        public String id() {
+            return id;
+        }
+
+        /**
+         * Get description.
+         *
+         * @return description
+         */
+        public String description() {
+            return description;
+        }
+    }
+
+    /**
+     * Validation.
+     */
+    public static final class Validation extends Input {
```

Comment:
This should be a block.
Also, instead of using `Validations` and `Validation`, why not call this class `Regex` and the enclosing class `Validation`.